### ISSUE TITLE
Vertex Shader (for CanvasItems) rotates to the wrong direction in Godot 3.2.4.

### ISSUE THREAD

**Issue by takaturre** [godotengine/godot#46895]
**Godot version:**
Godot 3.2.4 rc1-4. Works correctly in Godot 3.2.3 stable.

**OS/device including version:**
Windows 10, GLES3. (GPU: Nvidia GTX1080)

**Issue description:**
When creating a `VertexShader` (with just `VERTEX = VERTEX;` in the vertex function), and rotating the `CanvasItem` (eg. a `Sprite`), the rotation direction is flipped. See screenshot. (The 'editing boundary' of the Sprite rotates correctly, graphical content not.)

**Steps to reproduce:**
- Create a `Sprite`.
- Create a `ShaderMaterial` for it.
- Create a `Shader` script.
- Write into the script:
```
shader_type canvas_item;
void vertex() {
   VERTEX = VERTEX; // Uncommenting this, makes it work fine - as the shader is essentially bypassed.
}
```
![problem_screenshot](https://user-images.githubusercontent.com/16601926/110791313-bccb6780-827a-11eb-96b0-b87a4af680cd.JPG)

**Minimal reproduction project:**
[VertexShaderProblem.zip](https://github.com/godotengine/godot/files/6123092/VertexShaderProblem.zip)

**Comment by Calinou** [comment]
I can reproduce this on 3.2.4rc3, both with the GLES3 and GLES2 renderers. However, this doesn't occur when batching is disabled (both in GLES3 and GLES2).

cc @lawnjelly

**Comment by lawnjelly** [comment]
Ah this sounds like a simple polarity mishap lol. I'll take a look. :grinning:

**Comment by lawnjelly** [comment]
Fixed by #46898
